db/seeds.rb

```diff
@@ -174,126 +314,353 @@
+                    personal_statement_guidance: Faker::Lorem.paragraph(sentence_count: rand(5..10)),
+                    school_visits: Faker::Lorem.paragraph(sentence_count: rand(5..10)))
+end
+
+#### Expired
+
+4.times do
+  school = la_schools.sample
+
+  expired_vacancy = FactoryBot.build(:vacancy, :expired,
+                                     id: SecureRandom.uuid,
+                                     publisher_organisation: school,
+                                     publisher: publishers.sample,
+                                     organisation_vacancies_attributes: [{ organisation: school }],
+                                     about_school: Faker::Lorem.paragraph(sentence_count: rand(5..10)),
+                                     benefits: Faker::Lorem.paragraph(sentence_count: rand(1..3)),
+                                     job_title: Faker::Lorem.sentence[1...30].strip,
+                                     job_advert: Faker::Lorem.paragraph(sentence_count: rand(50..300)),
+                                     personal_statement_guidance: Faker::Lorem.paragraph(sentence_count: rand(5..10)),
+                                     school_visits: Faker::Lorem.paragraph(sentence_count: rand(5..10)))
+  expired_vacancy.save(validate: false)
+end
+
+### At multiple schools
+
+FactoryBot.create(:vacancy, :published, :at_multiple_schools,
+                  id: "e25a451f-9d73-4259-9142-64864f8c046d",
+                  publisher_organisation: local_authority_one,
+                  publisher: publishers.sample,
+                  organisation_vacancies_attributes: [
+                    { organisation: la_school_one },
+                    { organisation: la_school_two },
+                  ],
+                  about_school: Faker::Lorem.paragraph(sentence_count: rand(5..10)),
+                  benefits: Faker::Lorem.paragraph(sentence_count: rand(1..3)),
+                  job_title: Faker::Lorem.sentence[1...30].strip,
+                  job_advert: Faker::Lorem.paragraph(sentence_count: rand(50..300)),
+                  personal_statement_guidance: Faker::Lorem.paragraph(sentence_count: rand(5..10)),
+                  school_visits: Faker::Lorem.paragraph(sentence_count: rand(5..10)))
 
 Vacancy.index.clear_index
 Vacancy.reindex!
 
-jobseeker = Jobseeker.create(email: "jobseeker@example.com",
-                             password: "password",
-                             confirmed_at: Time.zone.now)
-
-FactoryBot.create(:job_application, :status_submitted,
-                  id: "6683c564-15e6-41af-ab44-7adf125f4c84",
-                  jobseeker: jobseeker,
-                  vacancy: physics_job)
+# Jobseekers
+
+## Create jobseeker account that users are accustomed to logging in with
+Jobseeker.create(email: "jobseeker@example.com", password: "password", confirmed_at: Time.zone.now)
+
+# Create extra jobseekers so each vacancy has multiple applications
+i = 1
+loop do
```

Comment:
```ruby
10.times.with_index
```

Reply:
Apparently you don't even need the with_index !

```
10.times do |i|
  blah blah 
end
```